src/Effects/Effects.UWP/Effects/Picker/PickerChangeColor.cs

```diff
@@ -1,22 +1,22 @@
 ﻿using Windows.UI.Xaml.Controls;
-using FormsCommunityToolkit.Effects.UWP.Effects;
 using Xamarin.Forms;
 using Xamarin.Forms.Internals;
 using Xamarin.Forms.Platform.UWP;
 using Windows.UI.Xaml.Media;
+using FormsCommunityToolkit.Effects;
+using PickerChangeColor = FormsCommunityToolkit.Effects.UWP.Effects.PickerChangeColor;
```

Comment:
It builds / works at my end... maybe best you tweak this to see how result is on your machine?